src/lib/profiles/vendor/nestlabs/device-description/NestProductIdentifiers.hpp

```diff
@@ -64,6 +64,7 @@ enum NestWeaveProductId
     kNestWeaveProduct_SDKSampleLightActuator     = 0xFE06,
     kNestWeaveProduct_SDKSampleButtonSensor      = 0xFE07,
     kNestWeaveProduct_SDKSampleMotionSensor      = 0xFE08,
+    kNestWeaveProduct_OneDeviceSDKLockExample    = 0xFE0B,
```

Comment:
Can we name this simply "SDKLockExample"?

Reply:
Just saw this comment. Yep, will do !

Reply:
Done !